PathOfTerraria.csproj

```diff
@@ -17,6 +17,9 @@
     <BuildCommand Condition=" Exists('/etc/NIXOS') ">steam-run dotnet</BuildCommand>
     <BuildCommand Condition=" '$(BuildCommand)' == '' ">dotnet</BuildCommand>
   </PropertyGroup>
+  <ItemGroup>
+    <None Remove="Localization\en-US\Mods.PathOfTerraria.Buffs.hjson" />
```

Comment:
This needs to be removed

Reply:
I wish we could remove the csproj from being updated, this stuff is going to happen pretty consistently. The linter wants it though, so...

Reply:
Yea VS does that and so does Rider. And I drives me crazy xD